README.md

```diff
@@ -519,6 +520,11 @@ Codeplay Software Ltd.
 Feel free to create an issue on the Github tracker to request features or
 report bugs.
 
+### Missing features
+
+Currently portBLAS is not complete, at [this link](doc/MissingFeatures.md) there is a list of the feature still missing.
+From input types support to new operators feel free to work on any of those.
```

Comment:
This sounds a bit redundant now, maybe consider adding it to the list of other documents below?